regulations/static/regulations/css/scss/partials/_reg_text.scss

```diff
@@ -18,13 +16,44 @@
         }
     }
 
+    .reference {
+        position: absolute;
+        margin-left: -1.5em;
+        visibility: hidden;
+    }
+
+    & > .reg-section > h2,
+    & > .reg-section > .paragraphs > p {
+        margin-left: -$spacer-2;
+        padding-left: $spacer-3;
+    }
+
+    h1 > .reference,
+    h2 > .reference {
+        margin-left: -1.2em;
+
+        @include screen-xxl {
+            margin-left: -1.4em;
+        }
+    }
+
+    :hover {
+        > .reference {
+            visibility: visible;
+        }
+    }
+
     .section-title {
-      color: $primary_color;
+        color: $primary_color;
+    }
+
+    .part-title {
+        margin: $spacer-5 0 $spacer-3;
     }
 
     h2 {
-       color: $black;
-       font-size: $base-font-size * 1.2;
+        color: $black;
```

Comment:
We're trying to avoid calling colors by names directly - in `application_settings.scss` there should be an abstracted variable we can call (i.e. `$color_primary`or similar rather than `$teal`) or you can add a new variable there and call it.

Reply:
`$black` has been changed to `$title_color` and updated where used.